ddtrace/tracer/span.go

```diff
@@ -66,6 +66,7 @@ type span struct {
 	TraceID  uint64             `msg:"trace_id"`          // identifier of the root span
 	ParentID uint64             `msg:"parent_id"`         // identifier of the span's direct parent
 	Error    int32              `msg:"error"`             // error status of the span; 0 means no errors
+	TopLevel bool               `msg:"top_level"`         // boolean indicating if the span is a top level
```

Comment:
I don't think this is used anywhere, and I think this might break things too. Let's remove it.